sphinx/source/users/issue_commands.rst

```diff
@@ -52,29 +52,23 @@ Checking for Commit
 
 Because of the decentralised nature of CCF, a request is committed to the ledger only once a number of nodes have agreed on that request.
 
-To guarantee that their request is successfully committed to the ledger, a user needs to issue a ``getCommit`` request, specifying the ``commit`` version received in the response. If CCF returns a ``global-commit`` greater than the ``commit`` version at which the ``LOG_record`` request was issued `and` that result ``commit`` is in the same ``term``, then the request was committed to the ledger.
+To guarantee that their request is successfully committed to the ledger, a user should issue a ``getTxStatus`` request, specifying the transaction version received in the response. The response may say the initial transaction is still pending global commit, has been globally committed, or has been lost due to a consensus leadership change (in which case the request should be resubmitted).
```

Comment:
Should we also replace most occurrences of "global commit" with "commit" on this page? With this new status RPC, a transaction is either `PENDING` or `COMMITTED`.

Reply:
Yes we should, but I've probably left a few 'global's for emphasis in this version. Its tricky to be precise while we still have mixed terminology, let's aim to replace all `term.commit` and remove local commit, then we can cleanly reduce global commit to commit.